kong/templates/nginx.lua

```diff
@@ -19,8 +23,22 @@ events {
 > end
 }
 
+> if wasm and wasm_modules_parsed and #wasm_modules_parsed > 0 then
+wasm {
+  shm_kv kong_wasm_rate_limiting_counters 12m;
```

Comment:
@locao is `shm_kv` something that can be injected via the conf_loader with a `nginx_wasm_*` value in kong.conf?

I think one of our internal integration docs references a toy ratelimiting filter which [likely] relies on this (citation needed: @hishamhm?), so I want to be able to update that doc with the proper instructions before removing this line.

Reply:
Yes to all 😄 

This is a left over entry from testing. The show-case rate-limiting relies on this entry, but it should be injected, not part of the default conf.

Reply:
@locao Nice! I've added a ticket so we remember to add those to the README and `test/` script in both showcase filters later.